modules/govuk_jenkins/templates/jobs/deploy_app_downstream.yaml.erb

```diff
@@ -53,14 +53,14 @@
 
           if [ "$LATEST_TAG_NAME" != "\"$TAG\"" ]; then
             echo "The TAG parameter does not match the latest release. Aborting."
-            exit 1
+            exit 0 # Don't alert about predictable failures
           fi
 
           if [ "$LATEST_TAG_SHA" != "$LATEST_MASTER_SHA" ]; then
             echo "The TAG to deploy is supserseded, or not on master. Aborting."
-            exit 1
+            exit 0 # Don't alert about predictable failures
           fi
-      - shell: |
```

Comment:
It might be worth splitting up the commit because there's lots of little changes in this PR, but the crucial one is this line here, which I completely missed the first time and didn't understand how this fixed the problem.

Reply:
Sure, it's a fair point - I was in a rush when I did this. I've split the change into 3 commits. Could you take another look?